packages/lisk-transaction-pool/test/check_transactions.ts

```diff
@@ -23,7 +23,9 @@ import transactionObjects from '../fixtures/transactions.json';
 import { wrapTransferTransaction } from './utils/add_transaction_functions';
 
 describe('checkTransactions', () => {
-	const transactions = transactionObjects.map(wrapTransferTransaction);
+	const transactions = transactionObjects
```

Comment:
Rather than adding the containsUniqueData property for all transactions, we should update the TransactionObject interface in TransactionPool class to have containsUniqueData property as optional. This is because the transactions fixtures contain other types of transactions as well, the change in this PR will later create conflicts in many other PRs.